docs/svsm.md

```diff
@@ -0,0 +1,135 @@
+# SVSM  Utility Script (Launch_SVSM.sh)
+
+This 'Launch_SVSM.sh'  utility script is a provisioning and test script that performs three tasks:
+
+1. Sets up an AMD EPYC CPU powered server by building host kernel, guest kernel, IGVM,  qemu, ovmf builds
+
+2. Direct boot launch a SNP enabled guest with qemu using IGVM file
```

Comment:
```suggestion
2. Direct boot launch a SNP enabled guest with SVSM using QEMU and a IGVM file.
```

Reply:
How about like this "Launch an SNP-SVSM-enabled guest using SVSM and IGVM-supported options in the QEMU command"